cabal-install/Distribution/Client/Init.hs

```diff
@@ -889,6 +889,12 @@ generateCabalFile fileName c =
                  Executable -> "Modules included in this executable, other than Main.")
               True
 
+     , fieldS "autogen-modules" (listField (otherModules c'))
```

Comment:
I don't think there's a need to include this field in the default `cabal init`; it's definitely a power-user case.